example-get-started/generate.sh

```diff
@@ -219,7 +218,7 @@ git push --force origin --tags
 
 Run these to drop and then rewrite the experiment references on the repo:
 
-git ls-remote origin "refs/exps/*" | awk '{print \$2}' | xargs -n 1 git push -d origin
+git ls-remote origin "refs/exps/*" | awk '{print \\$2}' | xargs -n 1 git push -d origin
```

Comment:
Is this right? It's different than https://github.com/iterative/example-repos-dev/pull/106/files#diff-f72bf81b747918d477ba9f4d2d4251111fbf04faf815d95716c13a093314a372L204-R222 . Maybe merge `main` to this branch instead.